### ISSUE TITLE
[FEATURE] New "preset" colors color ramp option

### ISSUE THREAD

**Issue by QGIS-feature-tracker** [qgis/QGIS-Documentation#1409]
Original commit: https://github.com/qgis/QGIS/commit/ec50cac2ef96d171bbb68e5765ce1df9c916f78f by nyalldawson

Allows use of a color ramp consisting of a list of selected colors.
Currently there's no way in QGIS to classify a renderer using
some list of colors you've previously selected. So you can modify
the colors manually after classifying, but that's a pain
if you're regularly using the same color scheme.

Basically, it's like the color brewer color ramp options but
allowing users to pick their own preset list of colors to use*

(Because Cynthia Brewer isn't the only cartographic color expert!)
